domain-model-assistant/Assets/Components/Scripts/Edge.cs

```diff
@@ -121,6 +100,35 @@ void createEdge()
         }
     }
 
+    public Vector3[] GetPositions() 
+    {
+        Vector3[] positions = new Vector3[line.positionCount];
```

Comment:
How many positions does a line have? If it is fixed, see if there is a better data structure that can be used here instead of an array